src/DynamoApplications/Properties/Resources.Designer.cs

```diff
@@ -8,18 +8,18 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace DynamoApplications.Properties
-{
-
-
+namespace DynamoApplications.Properties {
```

Comment:
Is the "using System" line an expected change?

Reply:
@reddyashish this file is autogenerated.